### PR TITLE
Quality of life changes

### PR DESCRIPTION
**Pull request by ficool2**
- Added viewmodels for the keycard, healthkit and grenade items. They are now also shown in thirdperson properly
- Added sound effect when trying to use something with insufficient access level, also added a class reaction for this
- Added charge up sound effect for Micro H.I.D.
- Teammates will now randomly play a class reaction if a fellow teammate dies nearby
- Health kit is now easier to use, only requires holding +attack. The viewmodel animation will give an indicator of progress
- Grenades can now teleport across elevators
- Fixed SCP 173 being seen through walls and getting stuck in vents

Balancing change: gun damage has been reverted to the old lower values. The high gun damage killed humans and SCPs too easily, and it caused people to be very trigger-happy with guns. Players have agreed it would be better if the gun damage was much lower like before

### CODE REVIEW

File: addons/sourcemod/scripting/scp_sf/scps/173.sp [hunk 1]
```diff
@@ -55,6 +55,9 @@ public bool SCP173_Create(int client)
 		SetEntProp(weapon, Prop_Send, "m_iAccountID", GetSteamAccountID(client, false));
 		SetEntPropEnt(client, Prop_Send, "m_hActiveWeapon", weapon);
 	}
+	
+	// no crouching
+	TF2Attrib_SetByDefIndex(client, 820, 1.0);
```
Comment (Batfoxkid) [inline addons/sourcemod/scripting/scp_sf/scps/173.sp:60]: This doesn't get reset on changing classes or round restart, probably better too have it on the weapon. There's also a way to fix crouching via `SetEntProp(client, Prop_Send, "m_bDucked", true);` before teleporting which I forgot to add.

**Comment by NextinMono** [comment]
👍

**Review by Batfoxkid — COMMENTED** [review]
Amazing

File: addons/sourcemod/scripting/scp_sf/scps/173.sp (hunk 1 above)
Comment (ficool2) [reply addons/sourcemod/scripting/scp_sf/scps/173.sp]: Fixed, tested and works fine on the weapon. I disabled crouching entirely for peanut as a tall statue shouldn't be able to fit inside a tiny vent

**Review by Batfoxkid — APPROVED** [review]

**Pull request merged**